ports/icu/portfile.cmake

```diff
@@ -103,5 +108,20 @@ endif ()
 # Prepare distribution
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/sbin)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/sbin)
+
+# Merge cmake configs
+file(COPY
+    DESTINATION ${CURRENT_PACKAGES_DIR}/share/icu
+    PATTERN ${CURRENT_PACKAGES_DIR}/debug/lib/cmake/*.cmake
+)
+file(COPY
+    DESTINATION ${CURRENT_PACKAGES_DIR}/share/icu
+    PATTERN ${CURRENT_PACKAGES_DIR}/lib/cmake/*.cmake
+)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/lib/cmake)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib/cmake)
+
 file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/icu RENAME copyright)
-file(WRITE ${CURRENT_PACKAGES_DIR}/share/icu/version "${VERSION_MAJOR}.${VERSION_MINOR}.0")
+file(WRITE ${CURRENT_PACKAGES_DIR}/share/icu/version "${ICU_VERSION_MAJOR}.${ICU_VERSION_MINOR}.0")

```

Comment:
All the `FOO_VERSION` bits were renamed to just be `VERSION` so this line shouldn't change.